src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -719,6 +720,8 @@ void uct_ib_mlx5_txwq_validate_always(uct_ib_mlx5_txwq_t *wq, uint16_t num_bb,
 #if HAVE_DEVX
 
 ucs_status_t uct_ib_mlx5_devx_create_qp(uct_ib_iface_t *iface,
+                                        uct_ib_mlx5_cq_t *send_cq,
+                                        uct_ib_mlx5_cq_t *recv_cq,
```

Comment:
maybe pass it as array?

Reply:
Seems like we cannot do it by a such way because `uct_rc_mlx5_devx_create_cmd_qp` passes RX CQ to both send and receive CQs before QP creation (check out [here](https://github.com/openucx/ucx/pull/8300/files#diff-f12c727872eb471ada737731e26c261d093dc0ea19feb77e1643e6bdc7267b4aR284))

Reply:
ok